ITP-MAY-25 | London | Chi Mbah | Onboarding | Wireframe | Sprint 1

Chi-Mb opened the pull request:
<!--

You must title your PR like this:

COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK

For example,

NW4 | Carol Owen | HTML-CSS-Module | Week1

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [x] I have tested my changes
- [ ] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)
- [ ] My changes meet the [requirements](./README.md) of this task

## Changelist

Resubmission for May Cohort

This pull request resubmits the original work from PR #161, originally submitted during the January cohort, as advised.

Briefly explain your PR.

## Questions

Ask any questions you have for your reviewer.

Chi-Mb reviewed (commented):
Needs to be reviewed

davidbrown2324 reviewed (commented):
Hi @Chi-Mb,

This is trivial, but it doesn't look like the numbered list is working in your markdown. Every section is section 1.

davidbrown2324 commented:
I am a new volunteer, and just finding the rhythm.

You have not used the PR Template checklist.

I am sure the code is good, but as a reviewer this leaves me a lot to check through.
I haven't looked at all your PRs, so if you were just busy that might be fine, but if you weren't sure how to use/ edit the check list it is worth finding out if you have time.

davidbrown2324 commented:
- [x] You have committed my files one by one, on purpose, and for a reason.

davidbrown2324 reviewed (commented):
- [x] You have tested you web code using [Lighthouse](https://programming.codeyourfuture.io/guides/testing/lighthouse)
- [x] You have used version control by committing often and pushing regularly to GitHub
- [x] You page scored 100% for accessibility

- [ ] The QR code did not work from my phone. The QR worked, but the page did not load on my phone. Perhaps I have done something wrong, or out of context.

Chi-Mb commented:
@davidbrown2324 Thanks for bringing this issue to my attention. I’ve updated the numbered section of my markdown file, and hopefully, this resolves the problem.

davidbrown2324 commented:
@Chi-Mb hello again. I am now able to review I think.

There is still an issue with the accessibility of your page.
I clicked on the "view detailed breakdown and full score reports" in your [bot review ](https://github.com/CodeYourFuture/Module-Onboarding/pull/161#issuecomment-2595270679)

I got this screen shot of the issue. Which means you are scoring 97 not 100 for accessibility.

![image](https://github.com/user-attachments/assets/fd11f183-855a-42cc-9d88-79264f3daa90)

Please take a look and see if you can fix this.

Chi-Mb commented:
@davidbrown2324 I checked Lighthouse and identified the issues. I didn’t follow the semantic rules, such as skipping from an h2 to an h5 without including the other headers, which I’ve now corrected. I also updated the footer text to a darker color for better contrast with the background. After making these changes, I ran another Lighthouse test, and my accessibility score is now 100. Could you please double-check on your end to confirm if the score has improved?

davidbrown2324 reviewed (approved):
I am approving this so you can move forward.

Your changes did improve the accessibility score to 100.
On my browser the performance score has now dropped, I am not sure why.
There are probably still improvements here if you have time, but I think you can proceed with the course. 

Well done for identifying and fixing the accessibility issues.

davidbrown2324 commented:
I am not sure how to run the netlify bot again? Was that automatic or was that something you triggered?
